tests/integration/test_definitions/multiple_backend/integration_tests.py

```diff
@@ -29,6 +29,7 @@
 
 cross_table_comparisons = [
     IntegrationTestFixture(
+        # Run locally via `pytest -v --mysql --postgresql --docs-tests -m integration -k "test_docs[cross_table_comparisons]" tests/integration/test_script_runner.py`
```

Comment:
Was this a stray comment?

Reply:
@Shinnnyshinshin I did not enter this comment, but it seems appropriate to me; am I missing an error in it?  Thanks!

Reply:
We have the instructions here: 
```
https://github.com/great-expectations/great_expectations/blob/7b4265942e3ce4e43799ccc5b4bc45db51605482/tests/integration/integration_test_fixture.py#L14
```
If we are doing it for this `IntegrationTestFixture` only, it feels out of place.